ui/components/app/assets/nfts/nft-details/nft-details.tsx

```diff
@@ -851,3 +897,11 @@ export default function NftDetails({ nft }: { nft: Nft }) {
     </Page>
   );
 }
+
+function NftDetails({ nft }: { nft: Nft }) {
+  const { chainId } = useParams();
+
+  return <NftDetailsComponent nft={nft} nftChainId={chainId ?? ''} />;
+}
```

Comment:
Had to make this additional wrapper to satisfy the storybook ci test. It didn't like me using `useParams` in storybook, so this wrapper allows me to bypass that while maintaining functionality.